### PR TITLE
Features/sma 84 readme

### PR DESCRIPTION
**Pull request by Aboudjem**
changed template for PR + README files (included @rhicc's [suggestion](https://github.com/bcnmy/biconomy-client-sdk/pull/267))

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
 
-**Test Configuration**:
-* Firmware version:
-* Hardware:
-* Toolchain:
-* SDK:
+## Checklist
 
-# Checklist:
+-   My code is neat and clean
```
Comment (tomarsachin2271) [inline .github/pull_request_template.md:25]: Why the checkbox is removed?
[ ]

File: assets/readme/biconomy-sdk.png
Comment (tomarsachin2271) [inline assets/readme/biconomy-sdk.png:1]: This needs to be updated.

Comment (tomarsachin2271) [reply assets/readme/biconomy-sdk.png:1]: Will provide a new diagram

File: README.md [hunk 2]
```diff
@@ -1,49 +1,51 @@
...
+- **Smart Accounts**: Enhance user experience with ERC6900 compliant accounts.
+- **Paymaster Service**: Enable third-party gas sponsorship. They can also be used to accept gas payment in ERC20 tokens from the user's smart accounts, with the paymaster managing the gas payment in native token.
+- **Bundler Infrastructure**: Ensure efficient and reliable transaction bundling.
+- **Backend Node**: Manage chain configurations and gas estimations.
 
-## Services
+## 📦 Packages
 
-<img width="1076" alt="Screenshot 2022-11-13 at 7 45 04 PM" src="https://user-images.githubusercontent.com/90545960/201531668-b616d0b7-d94a-4ee5-9e4a-709836f8dfc0.png">
+### Account
```
Comment (tomarsachin2271) [inline README.md:25]: We should also show a code snippet of how a SmartAccount can be used using BiconomySmartAccountV2 class and same goes for paymaster and bundler.

File: README.md [hunk 3]
```diff
@@ -25,13 +24,34 @@ The Biconomy SDK is your all-in-one toolkit for building decentralized applicati
...
+const bundler: IBundler = new Bundler({
+    bundlerUrl: '', // From Biconomy Dashboard
+    chainId: ChainId.POLYGON_MUMBAI,
+    entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
+});
+```
+
 ### Paymaster
 
-Acting as third-party intermediaries, Paymasters have the capability to sponsor gas fees for an account, provided specific predefined conditions are met. Additionally, they can accept gas payments in ERC20 tokens from users' smart accounts, with the Paymaster managing the conversion to native tokens for gas payment.
```
Comment (tomarsachin2271) [inline README.md]: Why this line is removed?

Additionally, they can accept gas payments in ERC20 tokens from users' smart accounts, with the Paymaster managing the conversion to native tokens for gas payment.

**Review by tomarsachin2271 — APPROVED** [review]

**Review by livingrockrises — APPROVED** [review]

**Comment by livingrockrises** [comment]
@Aboudjem min three reviewers / approvals is a lot

**Review by ankurdubey521 — APPROVED** [review]

**Comment by Aboudjem** [comment]
> @Aboudjem min three reviewers / approvals is a lot

it's only 2 not 3

**Comment by livingrockrises** [comment]
> > @Aboudjem min three reviewers / approvals is a lot
> 
> it's only 2 not 3

ok, all reviewers added must have added reviews?

**Comment by Aboudjem** [comment]
> > > @Aboudjem min three reviewers / approvals is a lot
> > 
> > 
> > it's only 2 not 3
> 
> ok, all reviewers added must have added reviews?

Screenshot 2023-09-11 at 6 43 13 PM

It might be one of these rules. 

I will also remove the "Require signed commits" rule for now.

**Comment by livingrockrises** [comment]
> > > > @Aboudjem min three reviewers / approvals is a lot
> > > 
> > > 
> > > it's only 2 not 3
> > 
> > 
> > ok, all reviewers added must have added reviews?
> 
> Screenshot 2023-09-11 at 6 43 13 PM
> It might be one of these rules.
> 
> I will also remove the "Require signed commits" rule for now.

can you resolve conflicts and help this get merged

**Pull request merged**